djangocms_versioning/admin.py

```diff
@@ -274,17 +274,41 @@ def get_preview_link(self, obj):
 
     get_preview_link.short_description = _("Preview")
 
+    def _get_field_modifier(self, request, modifier_dict, field):
+        method = modifier_dict[field]
+
+        def get_field_modifier(obj):
+            return method(obj, field)
+
+        get_field_modifier.short_description = field
+
+        return get_field_modifier
+
     def get_list_display(self, request):
         # get configured list_display
-        list_display = self.list_display
+        tuple_display = self.list_display
         # Add versioning information and action fields
-        list_display += (
+        tuple_display += (
             "get_author",
             "get_modified_date",
             "get_versioning_state",
             self._list_actions(request)
         )
-        return list_display
+        # Get the versioning extension
+        extension = _cms_extension()
+        modifier_dict = extension.add_to_field_extension.get(self.model, None)
+        if modifier_dict:
```

Comment:
There's a natural break in the code here to add this into a function after if modifier_dict.